examples/package.json

```diff
@@ -7,7 +7,7 @@
     "@datastax/astra-db-ts": "^0.1.2",
     "@pinecone-database/pinecone": "^1.1.2",
     "commander": "^11.1.0",
-    "llamaindex": "latest",
+    "llamaindex": "workspace:*",
```

Comment:
if use `workspace:*` here, it's probably not working with `npm`?

Reply:
yeah, let me use another way